buildSrc/src/main/kotlin/com/datadog/gradle/plugin/srpayloadupdate/SrPayloadOverrideTask.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.srpayloadupdate
+
+import org.gradle.api.tasks.Exec
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputDirectory
+
+open class SrPayloadOverrideTask : Exec() {
```

Comment:
same answer as above, also I like that I can add the `input path` relative to the project. What would be the drawback for using the plugin ?

Reply:
Basically it's the overhead of using the command line to start gradle to launch a command line. 
Right now everything you do in those task is using constants, which can still be done in the `local_ci.sh` file. You can still ensure that the assemble / install tasks are ran before calling adb. 

> I like that I can add the input path relative to the project
The input path is hardcoded, did you mean that you want to set it to use the module's package name instead? Is this comming in this PR or another one?

> for now I wanted to force it to always execute
If the adb command is ran from local_ci, it would always execute as there would not be any cache for it.